src/utils/bititer.h

```diff
@@ -52,6 +52,16 @@ inline unsigned long GetLowestBit(std::uint64_t value) {
 #endif
 }
 
+enum BoardTransform {
```

Comment:
It's not really nice to pollute the global namespace with simple names like `Flip`.
Maybe use `enum class` rather than plain `enum`?

Reply:
I'd rather add the word Transform to all of them than deal with 'enum class' for a flags type enum.
So I did that.